src/modules/jobs/job.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { JobService } from './job.service';
+
+@Controller('api/v1/jobs')
```

Comment:
There is global path prefix set. Remove api/v1/.  Only jobs is required

Also add docs